sources/api/datastore/src/lib.rs

```diff
@@ -147,7 +147,14 @@ pub trait DataStore {
         S: AsRef<str>,
     {
         for (key, value) in pairs {
-            trace!("Setting data key {}", key.name());
+            match committed {
+                Committed::Live => {
+                    info!("Setting data key {}", key.name());
+                }
+                _ => {
+                    trace!("Updating data key {}", key.name());
+                }
```

Comment:
Does it mean this?

```suggestion
            match committed {
                Committed::Live => {
                    info!("Committed data key {}", key.name());
                }
                state => {
                    trace!("Data key {} state changed to {:?}", key.name(), state);
                }
```

Reply:
Sure, that could be semantically more accurate. Will update shortly.